sycl/test-e2e/ESIMD/local_accessor_copy_to_from.cpp

```diff
@@ -0,0 +1,135 @@
+//==-- local_accessor_copy_to_from.cpp  - DPC++ ESIMD on-device test -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+//
+// TODO: Enable the test when GPU driver is ready/fixed.
+// XFAIL: opencl || windows || gpu-intel-pvc || gpu-intel-gen12
```

Comment:
This most likely will not work. @aelovikov-intel recently mentioned that vars like 'gpu-intel-gen12' are not set in SYCLOS.